controlpanel/api/migrations/0010_DATA_add_tools.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 2.2.4 on 2019-08-28 10:34
+
+import json
+import os
+import secrets
+
+from django.conf import settings
+from django.db import migrations
+
+
+TOOLS = [
+    {
+        'chart_name': 'rstudio',
+        'version': '2.1.1',
+        'name': 'RStudio',
+        'description': 'RStudio: 1.2.1335+conda, R: 3.5.1, Python: 3.7.1, patch: 3',
+        'values': {
+            'authProxy.cookieSecret': '<SECRET_TOKEN>',
+            'authProxy.auth0.domain': '%(domain)s',
+            'authProxy.auth0.client_id': '%(client_id)s',
+            'authProxy.auth0.client_secret': '%(client_secret)s',
+            'rstudio.secureCookieKey': '<SECRET_TOKEN>',
+        },
+    },
+    {
+        'chart_name': 'jupyter-lab',
+        'version': '0.4.3',
+        'name': 'Jupyter Lab',
+        'description': 'v0.6.7',
+        'values': {
+            'authProxy.auth0_domain': '%(domain)s',
+            'authProxy.auth0_client_id': '%(client_id)s',
+            'authProxy.auth0_client_secret': '%(client_secret)s',
+            'cookie_secret': '<SECRET_TOKEN>',
+        },
+    },
+    {
+        'chart_name': 'airflow-sqlite',
+        'version': '0.2.2',
+        'name': 'Airflow',
+        'description': '1.10.3',
```

Comment:
I wonder if `description` in these `Tool` records should be the actual description (e.g. a pseudo-user friendly description of the tool, possibly different than [the one in the helm chart](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/master/charts/rstudio/Chart.yaml#L2)) and there is room for a `appVersion` field with [the version of the tool](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/master/charts/rstudio/Chart.yaml#L5) (e.g. what we currently have in the `description`).

Don't know if it makes sense.

Reply:
Yeah, I thought about that. We can put anything in the description field in the database - these initial data migrations are only to keep the Control Panel working as it is (although there may be many different versions of Airflow, RStudio and Jupyter-Lab already deployed, so I think the version stuff may need to be switched off for now).

Reply:
> although there may be many different versions of Airflow, RStudio and Jupyter-Lab already deployed, so I think the version stuff may need to be switched off for now

What do you mean by "switch off the version stuff"? I didn't get to the upgrade logic that but I would imagine we'll use the `Tool.version` field to determine if a user tool is out-of-date no?

Reply:
This code may break because users already have versions of Airflow, RStudio and Jupyter that are not found in the database.

Reply:
I plan to have a better look at the rest but by "may break" do you mean that if a user has a version of a tool which is not in the CP DB it will not be shown in the Tools page at all?

I will probably mess with the DB locally later and see how it goes but ideally we may want to show the users' tools regardless of our latest/supported version (so that they can upgrade).